newdle/migrations/versions/20200914_1600_c161fdcfab19_add_notify_column_to_newdle_model.py

```diff
@@ -0,0 +1,32 @@
+"""Add notify column to Newdle model
+
+Revision ID: c161fdcfab19
+Revises: 93a638b96375
```

Comment:
```suggestion
Revises: 679eab4aab01
```

this is just metadata for humans, but let's be consistent :)